src/Hosting/TestHost/src/TestServer.cs

```diff
@@ -83,7 +83,12 @@ public IWebHost Host
         /// <remarks>
         /// Defaults to false.
         /// </remarks>
-        public bool AllowSynchronousIO { get; set; } = false;
+        public bool AllowSynchronousIO { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value that controls if <see cref="ExecutionContext"/> and <see cref="AsyncLocal{T}"/> values are preserved from the client to the server.
```

Comment:
"The default value is..."

Reply:
✔️ Also updating the other comment to move default value to `<summary>` since `<remarks>` don't show in IntelliSense (which is where you really want this info IMO).